netlify/functions/presence.js

```
@@ -4,80 +4,120 @@ const sql = neon();
         const activeUsers = await sql`
-          SELECT username, last_seen 
-          FROM user_presence 
+          SELECT
+            username,
+            last_seen,
+            is_typing,
+            typing_started_at
+          FROM user_presence
           WHERE last_seen > NOW() - INTERVAL '30 seconds'
           ORDER BY username ASC
         `;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)